packages/server/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+## Changelog
```

Comment:
I wouldn't bother with this changelog either since this package is just for local testing and demo purposes and isn't published.

Reply:
Fine. I think it's useful for developers to know the server changes for local development though.